downstream/modules/vault-aap/proc-vault-configuring-kv2-secret.adoc

```diff
@@ -0,0 +1,86 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="vault-configuring-kv2-secret"]
+
+= Configuring the `hashicorp.vault.kv2_secret` module
+
+[role="_abstract"]
+
+The `hashicorp.vault.kv2_secret` module performs Create, Update, and Delete operations on KV2 secrets through a unified interface.
```

Comment:
Just a note: Might want to add the acronym after the spelled out version of CRUD.